build/Common.props

```diff
@@ -18,6 +15,26 @@
     <NuGetAuditLevel>low</NuGetAuditLevel>
   </PropertyGroup>
 
+  <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+  <PropertyGroup>
+    <NetFrameworkMinimumSupportedVersion>net462</NetFrameworkMinimumSupportedVersion>
+    <NetStandardMinimumSupportedVersion>netstandard2.0</NetStandardMinimumSupportedVersion>
+
+    <!-- production TFMs -->
+    <TargetFrameworksForLibraries>net8.0;net6.0;netstandard2.0;$(NetFrameworkMinimumSupportedVersion)</TargetFrameworksForLibraries>
+    <TargetFrameworksForExporterStackdriver>net8.0;net6.0;$(NetFrameworkMinimumSupportedVersion)</TargetFrameworksForExporterStackdriver>
+    <TargetFrameworksForInstrumentationAspNetCore>net8.0;net6.0;netstandard2.0</TargetFrameworksForInstrumentationAspNetCore>
+    <TargetFrameworksForInstrumentationAWSLambda>net8.0;net6.0</TargetFrameworksForInstrumentationAWSLambda>
+
+    <!-- non-production TFMs -->
```

Comment:
Should non-production stuff go in `Common.nonprod.props`? e.g. `TargetFrameworksForTests` is to replace `SupportedNetTargets`.

Reply:
I can but in fact I reproduced what has been done in the main repo: https://github.com/open-telemetry/opentelemetry-dotnet/blob/60b7d9baf32ee437ba3e84f1886ba8917e8d9475/build/Common.props#L35